plugin.rb

```diff
@@ -192,6 +197,14 @@ def current_user
 
       DraftSequence.next!(editor.aliased_user, post.topic.draft_key)
     end
+
+    if post_revision.modifications.keys.include? "user_id"
+      original_user_id, revised_user_id = post_revision.modifications["user_id"]
+
+      if revised_user_id == alias_user_id
+        DiscourseStaffAlias::UsersPostsLink.create!(user_id: original_user_id, post_id: post.id)
+      end
```

Comment:
The question is if we want to restrict the `DiscourseStaffAlias::UsersPostsLink.user_id` to users in `SiteSetting.staff_alias_allowed_groups`, but the restriction feels entirely unnecessary...